src/Clean.sh

```
@@ -1,23 +1,23 @@
+# The commands in this script are specifically designed for macOS/Linux.
 
-# Runs dotnet clean for each csproj file
+# Runs the dotnet clean command for each .csproj file.
 for csproj in $(find . -name '*.csproj'); do
     dotnet clean $csproj
 done
 
-# Delete specified directories
-for dir in $(find . -type d \( -name "bin" -o -name "obj" -o -name "node_modules" -o -name "Packages" -o -name ".vs" -o -name "TestResults" -o -name "AppPackages" -o -name ".meteor" -o -name "App_Data" \)); do
+# Deletes specified directories
+for dir in $(find . -type d \( -name "bin" -o -name "obj" -o -name "node_modules" -o -name "Packages" -o -name ".vs" -o -name "TestResults" -o -name "AppPackages" -o -name ".meteor" \)); do
```

Shellcheck

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)